bezier-rs/docs/interactive-docs/wasm/src/lib.rs

```diff
@@ -138,4 +146,23 @@ impl WasmBezier {
 		let bezier_points: Vec<Vec<Point>> = self.0.reduce(None).into_iter().map(bezier_to_points).collect();
 		to_js_value(bezier_points)
 	}
+
+	pub fn arcs(&self, error: f64, max_iterations: i32, maximize_arcs: bool) -> Vec<JsValue> {
+		let options = ArcsOptions { error, max_iterations, maximize_arcs };
+		self.0
+			.arcs(options)
+			.iter()
+			.map(|sector| {
+				to_js_value(CircleSector {
+					center: Point {
+						x: sector.center.x,
+						y: sector.center.y,
+					},
+					radius: sector.radius,
+					start_angle: sector.start_angle,
+					end_angle: sector.end_angle,
+				})
+			})
+			.collect::<Vec<JsValue>>()
```

Comment:
You could collect it to `Vec<CircleSector>` first, and then convert it using `to_js_value`. The return type of the function would just be `JsValue`. Then you would only need to call `JSON.parse` once in `App.vue`, which I personally think is a bit cleaner, and more consistent with other placing in `wasm.rs` where we return lists (well, except `lookup_table`, which you could change at the same time 😉 )